src/main/java/com/example/bank/service/demo/entity/dto/TransactionRequestDto.java

```diff
@@ -0,0 +1,12 @@
+package com.example.bank.service.demo.entity.dto;
+
+import com.example.bank.service.demo.entity.Account;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class TransactionRequestDto {
+    private Account fromAccount;
+    private Account toAccount;
```

Comment:
how can we pass entity in requestDto?

please pass ids instead - never pass entities in RequestDto or return them in ResponseDto

Reply:
done